tests/tests_suite_test.go

```diff
@@ -121,24 +124,66 @@ func buildScheme() *runtime.Scheme {
 	return scheme
 }
 
-func installK3kChart(ctx context.Context, kubeconfig []byte) {
+func installK3SDocker(ctx context.Context) {
+	var (
+		err        error
+		kubeconfig []byte
+	)
+
+	k3sHostVersion, ok := os.LookupEnv("K3S_HOST_VERSION")
+	Expect(ok).To(BeTrue())
```

Comment:
If empty let's fallback to a known version, the same we have set in the Github Action: `v1.32.1+k3s1`. This is because if you run the test locally maybe you don't have the var set.

```suggestion
	k3sHostVersion := os.Getenv("K3S_HOST_VERSION")
	if k3sHostVersion == "" {
		k3sHostVersion = "v1.32.1+k3s1"
	}
```